zeebe/engine/src/main/java/io/camunda/zeebe/engine/state/authorization/DbAuthorizationState.java

```diff
@@ -47,6 +48,13 @@ public class DbAuthorizationState implements MutableAuthorizationState {
   private final ColumnFamily<DbLong, PersistedAuthorization>
       authorizationByAuthorizationKeyColumnFamily;
 
+  // authorization key -> owner type + owner id + resource type
+  private final DbForeignKey<DbCompositeKey<DbCompositeKey<DbString, DbString>, DbString>>
```

Comment:
👍 The inception here is amazing! 😄

Reply:
Yeah, that's why I had the idea of build the key merging the ownerType and ownerId. 😄